apps/testrunner/actions/phantomas_test.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+import logging
+
+import phantomas
+
+from . import Action
+from phantomas.results import Runs
+
+
+logger = logging.getLogger(__name__)
+
+
+class PhantomasTest(Action):
+    REQUIRED_PARAMS = (
+        'url',
+    )
+
+    def __init__(self, *args, **kwargs):
+        super(PhantomasTest, self).__init__(*args, **kwargs)
+
+    def run(self):
+        try:
+            phantomas_runner = phantomas.Phantomas(
+                url=self.params['url'],
+                runs=self.params['retries']
+            )
+            phantomas_result = phantomas_runner.run()
+            if hasattr(phantomas_result,'_runs'):
+                all_runs = phantomas_result._runs
+            else:
+                all_runs = [phantomas_result]
+            self.result[self._RESULT_NAME].extend([
```

Comment:
I would think about wrapping it into property at some time since we are going to use it quite often